src/python/pants/engine/streaming_workunit_handler.py

```diff
@@ -51,6 +61,23 @@ def get_observation_histograms(self):
         """
         return self._scheduler.get_observation_histograms()
 
+    def get_expanded_specs(self) -> Dict[str, List[str]]:
```

Comment:
Let's define a type alias here (at the very least) so we can caller can use that type alias too when they pass this around.